src/navability/services/Variable.jl

```diff
@@ -79,7 +87,11 @@ function getVariables(navAbilityClient::NavAbilityClient, client::Client)::Vecto
     return get(sessions[1],"variables",[])
 end
 
-function ls(navAbilityClient::NavAbilityClient, client::Client)::Vector{String}
+function listVariables(navAbilityClient::NavAbilityClient, client::Client)::Vector{String}
     variables = getVariables(navAbilityClient,client)
     return map(v -> v["label"], variables)
+end
+
+function ls(navAbilityClient::NavAbilityClient, client::Client)::Vector{String}
```

Comment:
Just FYI this can be a shortcut like:
```
ls(navAbilityClient::NavAbilityClient, client::Client)::Vector{String} = listVariables(navAbilityClient,client)
```

Reply:
Will do in the future